package-lock.json

```diff
@@ -2688,6 +2688,7 @@
           "resolved": "https://registry.npmjs.org/extend-shallow/-/extend-shallow-2.0.1.tgz",
```

Comment:
Looks like the changes included here are not needed, is one of this cases of different environments generating the same `package-lock.json`

Reply:
Talked to @sudheerDev about it. He asked me to keep the changes.